src/lab11.py

```diff
@@ -417,28 +419,43 @@ def __init__(self):
             self.BLUE_MASK = 0x00ff0000
             self.ALPHA_MASK = 0xff000000
 
+    def init_chrome(self):
+        self.chrome_font = get_font(20, "normal", "roman")
+        chrome_font_height = linespace(self.chrome_font)
+
+        self.padding = 5
+        self.tab_header_bottom = chrome_font_height + 2 * self.padding
+        self.addressbar_top = self.tab_header_bottom + self.padding
+        self.chrome_bottom = \
+            math.ceil(
+                self.addressbar_top + chrome_font_height + 2 * self.padding)
```

Comment:
I assume this doesn't change over the course of a run? I'd rather these be locals, not fields. (And `self.padding` would ideally by a global, maybe `CHROME_PADDING_PX`.)

Reply:
Padding could be a global. The rest can be local only if we are willing to recompute them in the *_bounds methods every time.